.github/workflows/ci.yaml

```
@@ -81,14 +91,48 @@ jobs:
   #unit-test:
   #runs-on: "ubuntu-latest"
 
-  setup-kubernetes-cluster:
+  k8s-cypress-tests:
```

The way this is currently set up makes this an end-to-end test, so maybe we should make that explicit in the job name
```suggestion
  e2e-testing:
```